### DISCUSSION TITLE
Caches: Web UI: Add option to delete all caches and a checkbox

### DISCUSSION THREAD

**Discussion by hfhbd** [community/community#53030] [Actions]
### Select Topic Area

Product Feedback

### Body

Currently, you can only delete one action cache at a time using the web UI.
Instead, please also add a delete all and an individual checkbox to delete multiple caches using the web UI at once.
Without it, deleting many caches is annoying using web UI. I am aware of the CLI extension, but I would also having the UI option too.

**Comment by juzerali** [comment]
Where is the option to delete the cache?
